Jenkinsfile

```diff
@@ -65,6 +65,9 @@ pipeline {
               withDockerContainer(image: 'couchdbdev/centos-6-erlang-18.3', args: '-e LD_LIBRARY_PATH=/usr/local/bin --user 0:0') {
                 sh '''
                   cwd=$(pwd)
+                  rm -rf /tmp/couchjslogs
+                  mkdir -p /tmp/couchjslogs
+                  export COUCHDB_IO_LOG_DIR=/tmp/couchjslogs
```

Comment:
You want this envvar in the `environment {}` block at the top of the file so you don't have to repeat yourself 10 times.